### ISSUE TITLE
Get rid of frightening warning

### ISSUE THREAD

**Issue by probonopd** [TheAssassin/AppImageLauncher#187]
Warnings should only be put on the screen if something dangerous is going to happen, or if something went wrong.

Normal users are frightened by warnings, and avoid software that puts out warnings, fearing that they might "break the computer".

![](https://user-images.githubusercontent.com/2480569/60642812-a40d7c00-9e20-11e9-83e5-18976f16af1d.png)

According to the [AppImage Design Principles](https://github.com/AppImage/AppImageKit/wiki/Design-principles#end-user-experience), we want to avoid unnecessary screens, clicks, and technobabble.
No one besides us knows what "integrating" even means. The Mac never ever asks something like this. It just works.

[Good design is as little design as possible](https://de.wikipedia.org/wiki/Dieter_Rams). I did not invent this ;-)

```
me@host:~$ /home/me/.local/lib/appimagelauncher-lite/appimagelauncher-lite.AppImage cli --version
ail-cli version 1.3.1 (git commit b9b466a), built on 2019-07-04 00:23:30 UTC

me@host:~$ cat .config/appimagelauncher.cfg 
[AppImageLauncher]
ask_to_move=false
destination=/isodevice/Applications
enable_daemon=true
```

**Comment by TheAssassin** [comment]
This application is not trying to enforce some weird Mac design failures to Linux desktops. It's a bridge to what Linux desktops currently *are* and tries to integrate AppImages with that nicely in a way that Average Joe can understand. It highlights AppImages advantages, but doesn't try to belittle the user.

Please stop acting in the issue tracker like AppImageLauncher Lite was the only ever use case for AppImageLauncher. No, it's the *secondary* and *less powerful* and therefore *worse* experience. It's not the recommended installation way, it's an alternative. Primary is and will always be the real installation into the system, which offers all the nice features.

You should always consider that there is a reason behind such messages, warnings etc. You really seem like you accuse us of incompetence as we add "unnecessary" things. That's just not the case, and it's an annoying attitude to work with. AIL Lite is brand new. We might have to adjust the feature set and maybe selectively deactivate some functionality to improve its UX. That doesn't mean dialogs or anything are unnecessary in general. Seriously, @probonopd... We're trying to cooperate with you to improve AIL Lite to be a better software, but due to what I mentioned above, this is just no fun at all... Do you *have* to make this harder than necessary?

That warning used to be justified. I explained in https://github.com/TheAssassin/AppImageLauncher/issues/188#issuecomment-508876146 that re-integration after an update used to be a necessary step in this helper application but now no longer is since AppImageLauncher is capable of doing it on launch now.

Therefore this is to be considered a duplicate of #188, in which we already agreed on removing the functionality. That warning is part of that stuff and will logically be removed with the other integration code.

TL;DR: duplicate of #188, see https://github.com/TheAssassin/AppImageLauncher/issues/188#issuecomment-508876146.

**Comment by probonopd** [comment]
> This application is not trying to enforce some weird Mac design failures to Linux desktops. 

The AppImage project is there to make the Linux desktop more like the Mac desktop. Has been its mission since more than 10 years.

> Please stop acting in the issue tracker like AppImageLauncher Lite was the only ever use case for AppImageLauncher.

You are asking me to deprecate `appimaged` in favor of AILL, that's the reason I am working to bring AILL to a state that would make that viable. I am not interested in any aspects of non-Lite AIL.

> You should always consider that there is a reason behind such messages, warnings etc.

I am going for a polished experience, and I am not going to bodge.

> AIL Lite is brand new. We might have to adjust the feature set and maybe selectively deactivate some functionality to improve its UX. That doesn't mean dialogs or anything are unnecessary in general.

Yes, I realize this and maybe I should have stated it more explicitly in every of my contributions to this project, but _everything_ I am doing in this project is regarding AILL.

**Comment by TheAssassin** [comment]
>> This application is not trying to enforce some weird Mac design failures to Linux desktops.
> 
> The AppImage project is there to make the Linux desktop more like the Mac desktop. Has been its mission since more than 10 years.

It's not my mission. It's yours. But I'm not obliged to any of this.

> I am going for a polished experience, and I am not going to bodge.
> Yes, I realize this and maybe I should have stated it more explicitly in every of my contributions to this project, but everything I am doing in this project is regarding AILL. Again, I am not interested in any aspects of non-Lite AIL.

Very great way of trying to convince people of your own opinions. Call them bodgers and tell them you don't care about anything that doesn't affect you.

That really helps in dragging attention to your issues. It really makes people work on those with priority. It's not like people get annoyed and ignore you and your issues then...
